toolchain/check/testdata/deduce/binding_pattern.carbon

```diff
@@ -2,9 +2,9 @@
 // Exceptions. See /LICENSE for license information.
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 //
+// INCLUDE-FILE: toolchain/testing/testdata/min_prelude/convert.carbon
 // TODO: Add ranges and switch to "--dump-sem-ir-ranges=only".
 // EXTRA-ARGS: --dump-sem-ir-ranges=if-present
-// INCLUDE-FILE: toolchain/testing/testdata/min_prelude/convert.carbon
```

Comment:
TBH I prefer keeping the INCLUDE-FILE after (i.e. just to keep things sorted, lacking a better ordering), I just haven't been commenting because I expect most of the TODOs to be removed. But, it feels a bit odd to me to be touching a file just to move it?

Reply:
Hm, my thought here was that almost every test should eventually have `INCLUDE-FILE` in it, so it's best to come first, as this would aid copying tests to make new tests, and aid in discoverability. And it makes it much easier to quickly visually scan a file and see if it has `INCLUDE-FILE` in it yet, and makes tests without it stand out more clearly, which I considered as desirable. It is true it's not alphabetical. WDYT?

Reply:
Yeah, `AUTOUPDATE` has to be last cuz it gets `TIP`s placed below it. The copyright and INCLUDE-FILE end up being the header, and AUTOUPDATE being the footer, and exceptional stuff goes in between. We'll never get them alphabetical since `AUTOUPDATE` has to come last. Idk why `EXTRA-ARGS` going first feels wrong wrong to me. I guess I would like to be somewhat consistent either way, and alphabetical isn't going to work anyway.